### PR TITLE
Update docs for running the repository mining script

### LINKED ISSUE #202: Improve documentation about how to use repository.py
I am facing this particular issue when running repository.py 
@marco-c  @ayush1999 

```
Traceback (most recent call last):
  File "bugbug/bugbug/repository.py", line 203, in <module>
    download_commits(args.repository_dir, two_years_and_six_months_ago)
  File "bugbug/bugbug/repository.py", line 129, in download_commits
    first_rev = get_rev(hg, date_from)
  File "bugbug/bugbug/repository.py", line 123, in get_rev
    return hg.log(revrange='pushdate("{}")'.format(date.strftime('%Y-%m-%d')), limit=1)[0].node.decode('utf-8')
  File "C:\ProgramData\Anaconda3\lib\site-packages\hglib\client.py", line 1102, in log
    out = self.rawcommand(args)
  File "C:\ProgramData\Anaconda3\lib\site-packages\hglib\client.py", line 263, in rawcommand
    raise error.CommandError(args, ret, out, err)
hglib.error.CommandError: (255, b'', b'hg: parse error: unknown identifier: pushdate\n(did you mean date?)')

```

### PR DESCRIPTION
**Pull request by ayush1999**
Closes #202

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ Run the `run.py` script to perform training / classification. The first time `ru
 
 1. Clone https://hg.mozilla.org/mozilla-central/.
 2. Run `./mach vcs-setup` in the directory where you have cloned mozilla-central.
+3. In order to run the repository mining script, you'll have to enable a few extensions. Add the following to your .hgrc file, under the extensions section:
```
Comment (marco-c) [inline README.md]: `In order to run the repository mining script` is redundant with the title of the section, you can remove it. Just say `Enable the pushlog, hgmo and mozext extensions. For example, if you are on Linux, add the following ...`

Comment (marco-c) [reply README.md]: Replace `.hgrc` with `~/.hgrc`.

File: README.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ Run the `run.py` script to perform training / classification. The first time `ru
...
+    - pushlog = /PATH/version-control-tools/hgext/pushlog
+    - hgmo = /PATH/version-control-tools/hgext/hgmo
+    - mozext = /PATH/version-control-tools/hgext/mozext
+
+    where PATH is  `~/.mozbuild`.
```
Comment (marco-c) [inline README.md]: so you can remove this

File: README.md [hunk 3]
```diff
@@ -43,6 +43,11 @@ Run the `run.py` script to perform training / classification. The first time `ru
 
 1. Clone https://hg.mozilla.org/mozilla-central/.
 2. Run `./mach vcs-setup` in the directory where you have cloned mozilla-central.
+3. Enable the pushlog, hgmo and mozext extensions. For example, if you are on Linux, add the following to the `~/.hgrc` file:
+    - pushlog = ~/.mozbuild/version-control-tools/hgext/pushlog
```
Comment (marco-c) [inline README.md]: Sorry, one last comment. Somebody could be confused and copy the whole thing from here, so don't use - but wrap everything in ```.

**Comment by ayush1999** [comment]
@marco-c please review.

**Pull request merged**